packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataPersister/AbstractContextDataPersister.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\DataPersister;
+
+use ApiPlatform\Core\DataPersister\ContextAwareDataPersisterInterface;
+
+abstract class AbstractContextDataPersister implements ContextAwareDataPersisterInterface
+{
+    protected $decorated;
```

Comment:
We still need the PHPDoc on the properties as we're not type hinting them 😄 
The PHPDoc blocs are not required only when we can type hint the arguments/properties.

Reply:
And this doesn't need to be protected, it should be private.